test.js

```diff
@@ -8,5 +8,5 @@ test(t => {
 	t.is(m('darwin', '99.0.0'), 'macOS');
 	t.is(m('linux', '3.13.0-24-generic'), 'Linux 3.13');
 	t.is(m('win32', '5.1.2600'), 'Windows XP');
-	t.is(m('win32'), 'Windows');
+	t.is(m('win32'), 'Windows 98');
```

Comment:
This is not correct. It should continue to be `Windows`.

Reply:
I think the test should be fixed: i first changed the line without even looking at this.

But now that i'm looking i've found this: when i run the tests on linux, the effect is the release is fetched from the current os (in my case Fedora), so the effective call to winRelease in my case is like this

```
		id = winRelease('4.18.13-200.fc28.x86_64');
```
That is very wrong!

The fix should be on the test, may be passing a valid windows release number? And testing for the real os name, not just the generic "Windows" value as it was.

Reply:
If it's not on the current operation system and release is not passed, it should just return the OS name without version number, like before.